numpy/_core/fromnumeric.py

```diff
@@ -3572,8 +3573,8 @@ def size(a, axis=None):
     ----------
     a : array_like
         Input data.
-    axis : int, optional
-        Axis along which the elements are counted.  By default, give
+    axis : None or int or tuple of ints, optional
+        Axis or axes along which the elements are counted.  By default, give
         the total number of elements.
```

Comment:
Ah, this should also have a `.. versionchanged::` directive (you can search for similar ones it would be 2.4).
(Somehow, I don't care too much about the release note, but maybe best even if rather minor.)